lib/hexpm_web/templates/dashboard/audit_log/index.html.eex

```diff
@@ -0,0 +1,39 @@
+<div class="row">
+  <div class="col-sm-3">
+    <%= render DashboardView, "_sidebar.html", assigns %>
+  </div>
+
+  <div class="col-sm-9">
+    <div class="panel panel-default">
+      <div class="panel-heading">Recent activities</div>
+
+      <div class="panel-body">
+        <table class="table table-striped">
+          <tr>
+            <th class="col-sm-3" scope="col">Date</th>
+            <th scope="col">Activity</th>
+          </tr>
+
+          <%= for audit_log <- @audit_logs do %>
+            <tr>
+              <td class="col-sm-3"><%= pretty_date(audit_log.inserted_at, :short) %></td>
+              <td><%= humanize_audit_log_info(audit_log) %></td>
+            </tr>
+          <% end %>
+        </table>
+
+        <%=
+        render HexpmWeb.LayoutView,
+               "_pagination.html",
```

Comment:
Any recommendations for a better template name?

`_pagination` feels too vague. 
`_pagination_nav` feels too narrow.

Reply:
`_pagination` is fine imo.

Reply:
Good job extracting it btw.